eahub/profiles/api/serializers.py

```diff
@@ -0,0 +1,130 @@
+from enumfields.drf import EnumSupportSerializerMixin
+from rest_framework import serializers
+from rest_framework.relations import PrimaryKeyRelatedField
+
+from eahub.profiles.models import (
+    Profile,
+    ProfileTag,
+    ProfileTagType,
+    ProfileTagTypeEnum,
+)
+
+
+class TagTypeSerializer(EnumSupportSerializerMixin, serializers.ModelSerializer):
+    class Meta:
+        model = ProfileTagType
+        fields = [
+            "type",
+        ]
+
+
+class TagSerializer(serializers.ModelSerializer):
+    types = TagTypeSerializer(many=True)
+
+    class Meta:
+        model = ProfileTag
+        fields = [
+            "pk",
+            "name",
+            "types",
+            "count",
+        ]
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    tags_generic = TagSerializer(many=True, required=False)
```

Comment:
Probably to be discussed with product, but what's the purpose of the generic tag?

Reply:
the idea of mine was to unblock users - eg if somebody thinking that `couchsurfing` tag is a must, now instead of squeezing it into something nonsensual as "expertise area" they can add a tag as `Helping with couchsurfing" or "Looking for couchsurfing". After that we can notice those tags, analyze the demand and consider creating a better organized field or a set of fields.

I would say if we were god-like human beings who knew everything about our users' desires we wouldn't need a field as `tags_generic`

Reply:
I'm planning to import the values from EAG "Affinity" field, eg "People managers", "Earning to give", "Parenting", etc